src/main/java/pl/chojnacki/lesson5/homework/task5/Bicycle.java

```diff
@@ -0,0 +1,14 @@
+package pl.chojnacki.lesson5.homework.task5;
+
+public class Bicycle implements Vehicle {
+
+    @Override
+    public void startEngine() {
+        throw new UnsupportedOperationException("Bicycle does not have engine");
+    }
+
+    @Override
+    public void stopEngine() {
+        throw new UnsupportedOperationException("Bicycle does not have engine");
```

Comment:
A co z tym? Czy to nie śmierdzi trochę, że rower ma metody startEngine()/stopEngine() mimo, że nie ma silnika? ;)